Error: Unknown LLM provider: ollama.

I've followed the [documentation](https://chunkhound.github.io/configuration/#llm-configuration-code-research) for ollama LLM provider setup and got this error:

```
root@chunkhound:~/docs# chunkhound research "Does it work?"
09:32:52 | WARNING  | Using rerank_format='auto' may cause first request to fail if format guess is wrong. For production use, explicitly set rerank_format to 'cohere' or 'tei'.
[ERROR] LLM provider setup failed: Unknown LLM provider: ollama. Available providers: openai, claude-code-cli, codex-cli
[INFO] Configure an LLM provider via:
1. Create .chunkhound.json with llm configuration, OR
2. Set CHUNKHOUND_LLM_API_KEY environment variable
```


I managed to get it work by setting `"provider": "openai"` and exporting the environment variable `export OPENAI_API_KEY=whatever`, but probably either the documentation or the code should be updated.


I'm using this chunkhound version:
```
root@chunkhound:~# chunkhound --version
chunkhound 4.0.1
```

--- a/site/src/pages/docs/configuration.md
+++ b/site/src/pages/docs/configuration.md
@@ -414,6 +414,10 @@ No `rerank_url` is needed — it is auto-derived from `base_url`.
 
 LLM config:
 
+> **Migration note:** Do not set `llm.provider` to `"ollama"`.
+> ChunkHound treats Ollama as an OpenAI-compatible endpoint, so use
+> `provider: "openai"` with the Ollama `base_url` and an explicit `model`.
+
 ```json
 {
   "llm": {

